### PR TITLE
Support for absolute path asset imports

### PR DESCRIPTION
**Pull request by fabis94**
Related to this issue: https://github.com/sormy/rollup-plugin-smart-asset/issues/18

### CODE REVIEW

File: src/smartAsset.js [hunk 1]
```diff
@@ -182,7 +182,8 @@ export default (initialOptions = {}) => {
         return
       }
 
-      const id = importer ? join(dirname(importer), source) : source
+      const createId = (importer, source) => isAbsolute(source) ? source : join(dirname(importer), source)
+      const id = importer ? createId(importer, source) : source
```
Comment (sormy) [inline src/smartAsset.js:186]: I suggest to simplify code to:

```
const id = importer && !isAbsolute(source) ? join(dirname(importer), source) : source
```

Please also add a unit test.

Comment (fabis94) [reply src/smartAsset.js]: Added tests

**Comment by fabis94** [comment]
Added unit test

**Comment by sormy** [comment]
Thanks for contribution. The new version has been published v2.2.1.

**Pull request merged**